syncd/SaiSwitch.cpp

```diff
@@ -1002,29 +1017,19 @@ void SaiSwitch::onPostPortCreate(
         }
     }
 
-    SWSS_LOG_NOTICE("discovered %zu new objects (including port) after creating port VID: %s",
-            discovered.size(),
-            sai_serialize_object_id(port_vid).c_str());
+    SWSS_LOG_NOTICE("discovered %zu new objects (including port) after creating %zu ports",
+            discovered.size(), count);
 
     m_discovered_rids.insert(discovered.begin(), discovered.end());
 
-    SWSS_LOG_NOTICE("putting ALL new discovered objects to redis for port %s",
-            sai_serialize_object_id(port_vid).c_str());
+    std::vector<sai_object_id_t> rids{discovered.begin(), discovered.end()};
 
-    for (sai_object_id_t rid: discovered)
-    {
-        /*
-         * We also could thing of optimizing this since it's one call to redis
-         * per rid, and probably this should be ATOMIC.
-         *
-         * NOTE: We are also storing read only object's here, like default
-         * virtual router, CPU, default trap group, etc.
-         */
```

Comment:
is this operation now ATOMIC operation that you skipped this comment ?

Reply:
Re-added comment. redis pipeline is not atomic